### PR TITLE
Fix reduce-labels in BEIT Fast Image Processor

### PR DESCRIPTION
**Pull request by simonreise**
# What does this PR do?

When running `BeitImageProcessorFast` with `do_reduce_labels=True`, it not only reduces labels in segmentation map, but also applies `reduce_label` to the image itself

`reduce_label` should be applied only to segmentation maps

### Test

```py
import transformers
import torch


processor = transformers.BeitImageProcessorFast(
    do_resize=False, 
    do_center_crop=False, 
    do_rescale=False, 
    do_normalize=False, 
    do_convert_rgb=False,
)

image = torch.zeros([1, 3, 256, 256])

segmap = torch.zeros([1, 256, 256]) 

batch = processor.preprocess(
    images = image, 
    segmentation_maps = segmap, 
    return_tensors="pt", 
    do_reduce_labels=True,
)
```

pixel_values will be 255, should be 0.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@yonigozlan

### CODE REVIEW

File: src/transformers/models/beit/image_processing_beit_fast.py [hunk 1]
```diff
@@ -137,6 +137,16 @@ def _preprocess(
         processed_images = torch.stack(processed_images, dim=0) if return_tensors else processed_images
         return processed_images
 
+    def _preprocess_images(
+        self,
+        images,
+        **kwargs,
+    ):
+        """Preprocesses a single image."""
```
Comment (yonigozlan) [inline src/transformers/models/beit/image_processing_beit_fast.py:145]:
```suggestion
        """Preprocesses images."""
```
If you could also change the one for segmentation maps it would be great :)

**Review by yonigozlan — APPROVED** [review]
Thanks for catching that! Just a small nit on the docstring, otherwise LGTM!
